src/pages/AboutPage.tsx

```diff
@@ -1,8 +1,59 @@
-import React from "react";
+import React, { useState } from "react";
+import Button from "@material-ui/core/Button";
 import { Header } from "components/Header";
 
 export const AboutPage: React.FC = () => {
+  const [aboutData, setAboutData] = useState({
+    server: "openrainbow.com",
+    telephonyVersion: "unknown",
+    systemIdentifier: "PBX7467-...",
+    CopyrightYear: "2021",
```

Comment:
I like that you put this info in `state` because when we add the real app logic, we need to get this info from a remote server 👍